scipy/special/tests/test_gammainc.py

```diff
@@ -53,6 +55,13 @@ def test_x_zero(self):
         a = np.arange(1, 10)
         assert_array_equal(sc.gammainc(a, 0), 0)
 
+    def test_nan_at_a_and_x_0(self):
+        assert np.isnan(sc.gammainc(0,0))
```

Comment:
You should be able to just add `(0, 0)` to the `INVALID_POINTS` list above.

Reply:
done